src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
 // write code here
+
+function convertNumber(number) {
+  let final = number;
+
+  let final2 = final.slice(1).split(',').join('')
```

Comment:
use more declarative names for variables final final2 copy3 copy4 are meaningless

Reply:
done